### PR TITLE
Ensure rates are taken only from push queues

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/3582/

File: AdminServer/appscale/admin/push_worker_manager.py [hunk 1]
```diff
@@ -139,8 +139,9 @@ def _write_worker_configuration(self, queue_config):
       rates = {'default': '5/s'}
     else:
       queues = json.loads(queue_config)['queue']
-      rates = {queue: queues[queue]['rate'] for queue in queues
-               if 'mode' not in queue or queue['mode'] == 'push'}
+      rates = {
+        queue_name: queue['rate'] for queue_name, queue in queues.items()
+        if 'mode' not in queue or queue['mode'] == 'push'}
```
Comment (jeanleonov) [inline AdminServer/appscale/admin/push_worker_manager.py]: But this change doesn't modify content of resulting dict - old version and new are equal.
Does PR name correspond to it's content?

Comment (jeanleonov) [inline AdminServer/appscale/admin/push_worker_manager.py]: Ohhh.. now I see - there was a bug.

**Comment by cdonati** [comment]
Yes, I forgot to mention it in the description. For other reviewers, the previous code was looking for 'mode' in the queue name rather than the queue details.

**Pull request merged**